source/Calamari/Scripts/AzureContext.sh

```diff
@@ -40,10 +40,10 @@ function setup_context {
 
         echo "Azure CLI: Authenticating with Service Principal"
         loginArgs=()
-        loginArgs+=("-u $Octopus_Azure_ADClientId")
         # Use the full argument because of https://github.com/Azure/azure-cli/issues/12105
-        loginArgs+=("--password $Octopus_Azure_ADPassword")
-        loginArgs+=("--tenant $Octopus_Azure_ADTenantId")
+        loginArgs+=("--username=$Octopus_Azure_ADClientId")        
+        loginArgs+=("--password=$Octopus_Azure_ADPassword")
```

Comment:
Did you find out if we need to escape double/single quotes in bash?

Reply:
@johnsimons I tested it using passwords with double quotes and it didn't have any issues with argument parsing so I believe it's ok, seems like using the '=' just means it takes everything as is up to the end of the argument.

Reply:
How about if the password has spaces?

Reply:
Yeah that's a good thought, it prob would break I think, will prob need to wrap in some form of quotes like powershell and escape, i'll test and add handling. Thanks.

Reply:
So I've tested now with spaces and it does cause issues in the parsing of arguments, but the fix isn't actually to quote each argument it seems as this seems to make the quotes be included in the value of the argument and does nothing for handling of spaces. The fix is to put double quotes in the script around the passing of the loginArgs array. I'm not really sure why this behaviour is what it is, I've found some references to this such as https://forum.keyboardmaestro.com/t/how-to-escape-spaces-etc-in-when-passing-variable-to-shell/4386/6 but it seems to work when I've tested it both by running the script locally and properly in a Calamari build.

Reply:
@johnsimons I believe all changes should be captured now, would you mind giving this a final look please?